packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/components/SideBarFooter.astro

```diff
@@ -1,25 +1,18 @@
 ---
----
-<div
-    class="block sticky pointer-events-none bottom-10 bg-base-200 justify-center h-12 [mask-image:linear-gradient(transparent,#000000)]"
->
-</div>
+import type { StudioCMSSocials } from "../layouts/studiocms-socials";
 
+type Props = { studioCMSSocials: StudioCMSSocials };
+const { studioCMSSocials: { github: githubURL } } = Astro.props as Props;
```

Comment:
it does infer... but i noticed this also helps when the language server crashes... so i've been adding it in when i needed it instead of restarting the Language server